app/components/signatures/create-sign-flow.js

```diff
@@ -8,6 +8,8 @@ import { TrackedArray } from 'tracked-built-ins';
 import { startOfDay } from 'date-fns';
 import CONSTANTS from 'frontend-kaleidos/config/constants';
 
+const MAX_EMAIL_DISPLAY_LENGTH = 55;
+
```

Comment:
unused constant. (if we would use it, I would argue putting it in the `constants.js` or `config.js`